perf/benchmark/templates/mtls.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: authentication.istio.io/v1alpha1
+kind: Policy
```

Comment:
Policy and destinationrule was conditional in the previous template.
Why is it unconditional here?

Reply:
Since we are sending traffic from fortioclient to fortiopod, single direction. We only need to enforce this to fortioclient pod. Which the `name` and `host` is `fortioserver` for it to take effect.